### ISSUE TITLE
[4.4-overfitting] which model is better? (original model vs L2-regularized model)

### ISSUE THREAD

**Issue by songlinhit** [fchollet/deep-learning-with-python-notebooks#57]
![image](https://user-images.githubusercontent.com/31485958/42533752-c767030c-8458-11e8-95a9-eda633f07bed.png)
This example illustrates the impact of L2 regulation. I can see that the L2-regularied model gets overfitting slowly comparing with the original model. But the peak of validation loss of L2-regularied model is greater than that of the original model. Thus, for this example, the original model is still better than L2-regulaized model, right?

**Comment by RorisangSitoboli** [comment]
I see that the validation loss profile for the L2 regularized model peaks at around 0.4 in the figure while the standard model peaks at 0.8. The validation loss axis (y axis) is in positive units. From the plot you can also tell that both models start overfitting at 2.5 epochs but the standard model clearly performs poorly (far steeper gradient) after that compared to the regularized model. This must mean the regularized model performs far better.
